public/manifest.json

```diff
@@ -3,37 +3,59 @@
   "name": "weather-10kb",
   "icons": [
     {
+<<<<<<< HEAD:manifest.json
```

Comment:
It looks like there was a merge conflict that didn't get fully resolved. Recommend removing the `<<<HEAD...` errors